monai/transforms/io/dictionary.py

```diff
@@ -51,6 +51,10 @@ class LoadImaged(MapTransform):
         - Current default readers: (nii, nii.gz -> NibabelReader), (png, jpg, bmp -> PILReader),
           (npz, npy -> NumpyReader), (dcm, DICOM series and others -> ITKReader).
 
+    Please note that for png, jpg, bmp format images, `PILReader` will swap axis 0 and 1 after
+    loading the array because the `HW` definition for non-medical specific file formats in PIL is
```

Comment:
~in PIL~

Reply:
updated in the latest commit.

Thanks.